package.json

```diff
@@ -71,7 +72,8 @@
       ],
       "fxa-settings": [
         "fxa-shared",
-        "fxa-react"
+        "fxa-react",
+        "fxa-content-server"
```

Comment:
im not sure `modules-to-test.js` can handle cycles. it might

Reply:
Looks like it since these are stored in a set, https://github.com/mozilla/fxa/blob/20a0acf8b0d864702afb6d7c052f20a7f9ced526/.circleci/modules-to-test.js#L32.